main.tf

```diff
@@ -90,6 +90,28 @@ resource "equinix_metal_spot_market_request" "join_spot_request" {
   }
 }
 
+data "equinix_metal_device" "seed_device" {
+   device_id = data.equinix_metal_spot_market_request.seed_req.0.device_ids[0]
+}
+
+resource "equinix_metal_vlan" "vlans" {
+  count = var.num_of_vlans
+  project_id = data.equinix_metal_project.project.project_id
+  facility = var.facility
+}
```

Comment:
Could we also add a description? I'd recommend something like "VLAN for ${var.hostname_prefix}"

Reply:
Got it.